src/launcher.ts

```diff
@@ -147,77 +137,24 @@ export default class VSCodeServiceLauncher extends ChromedriverServiceLauncher {
             throw new Error(`No key "${VSCODE_CAPABILITY_KEY}" found in caps`)
         }
 
-        if (versionsFileExist) {
```

Comment:
Did you want to keep **versions.txt** at all @christian-bromann?

This looks really close. I think I've got it working/failing to start locally. That's a win because ChromeDriver on Linux on ARM isn't officially supported or downloaded.

Reply:
If you can confirm how to handle the cache, and that you wanted to set the `browserName = 'chrome' and `browserVersion` to the version of Chromium bundled with VS Code, then I think I've got most of it done.

The pipeline still fails but I suspect it's because the code building the download links from Puppeteer only supports a 3 digit version number, not a 4 digit like we extract from the manifest. :thinking:

Reply:
Awesome, thanks for looking into this. Mind raising a new PR and we take it from there?

Caching for Chromedriver is handled by WebdriverIO. It doesn't cache the VS Code binary though which I am not sure will become a problem. I've implemented this with the idea in mind that test should work no matter if the user has an internet connection or not. That said, I am ok with moving forward without that feature and raise an issue to re-add it back.

Reply:
I'll open a PR when I find a moment.

I believe we could keep with your original intent with the cache if we pivot to store the chromium version instead of chromium and let WebdriverIO handle Chromium/ChromeDriver.

Since I haven't got VS Code to open yet, would you happen to know if we'll want to use ChromeDriver or can we use Chrome for Testing to interact with VS Code? :thinking:

Reply:
Chrome for Testing and Chromedriver are different things. The first is the browser the other the driver. For automating VS Code we need Chromedriver and download the desired VS Code version. VS Code is an Electron application and "acts" as a browser here.

Reply:
Seems like I don't have to touch the `browserName` anyways. :tada: